### PR TITLE
Nginx: refactor integration

### PR DESCRIPTION
**Pull request by mtojek**
## What does this PR do?

This PR refactors the Nginx integration according to the migration action items:

- [x] Fields cleanup (use ECS ones)
- [x] Import ingress_controller dashboards
- [x] Verify ingress_controller dashboards work correctly
- [x] Update Docker image to latest version
- [x] Write system tests
- [x] Documentation: write a paragraph on how to prepare Nginx instances to expose metrics (server status)
- [x] Update the sample event for stubstatus dataset
- [x] Save sample events for other dataset

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md#tips-for-building-integrations) and this pull request is aligned with them.
- [x] I have verified that all datasets collect metrics or logs.

## How to test this PR locally

Boot up the Nginx service.
Verify if dashboards shows data.

## Related issues

- https://github.com/elastic/integrations/issues/373

### CODE REVIEW

**Comment by mtojek** [comment]
I've verified ingress controller dashboards, they seem to be working. Further works on dashboards (refactoring) will be performed in the followup issue.

Screenshots:

screencapture-localhost-5601-app-dashboards-2020-12-03-13_21_37
screencapture-localhost-5601-app-dashboards-2020-12-03-13_21_53

File: packages/nginx/_dev/deploy/variants.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+variants:
+  v1:
+    SERVICE_VERSION: 1.19.5
```
Comment (ChrsMark) [inline packages/nginx/_dev/deploy/variants.yml:3]: Watching this, does not look consistent with what we mention in `compatibility` section at https://github.com/elastic/integrations/pull/430/files#diff-a5eb205b220d180aec6ede43fd0a591a964a441d21c4bc69c22d50bdd184e834R8, or do I miss anything?

Maybe we need to tune the doc's section?

**Review by ChrsMark — COMMENTED** [review]
LGTM! Just one question about supported versions.

File: packages/nginx/_dev/deploy/variants.yml (hunk 1 above)
Comment (mtojek) [reply packages/nginx/_dev/deploy/variants.yml:3]: Good catch! I adjusted the README file.

**Review by ChrsMark — APPROVED** [review]
Thanks!

**Pull request merged**